msipackage/CMakeLists.txt

```diff
@@ -1,16 +1,30 @@
 set(BIN ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/${CMAKE_BUILD_TYPE})
+
+# For pipeline builds, use PACKAGE_INPUT_DIR if specified (contains signed binaries)
+# For local builds, use bin directory
+if(DEFINED PACKAGE_INPUT_DIR)
+    message(STATUS "Using signed binaries from ${PACKAGE_INPUT_DIR} for MSI packaging")
+else()
+    set(PACKAGE_INPUT_DIR ${BIN})
+    message(STATUS "Using binaries from ${BIN} for MSI packaging")
```

Comment:
nit: I'd have a small preference to only log this when it's overridden, so it doesn't show up every time when we run local builds